src/cantools/database/can/formats/sym.py

```diff
@@ -775,13 +775,18 @@ def _get_enum_name(signal: Signal) -> str:
 def _dump_choice(signal: Signal) -> str:
     # Example:
     # Enum=DPF_Actv_Options(0="notActive", 1="active", 2="rgnrtnNddAtmtcllyInttdActvRgnrt", 3="notAvailable")
+    nl = '\n'
+
     if not signal.choices:
         return ''
 
     enum_str = f'Enum={_get_enum_name(signal)}('
     is_first_choice = True
     for choice_number, choice_value in signal.choices.items():
-        enum_str += f'{", " if not is_first_choice else ""}{choice_number}="{choice_value}"'
+        if choice_number % 10 == 0 and not choice_number == 0:
+            enum_str += f'{","+nl if not is_first_choice else ""}{choice_number}="{choice_value}"'
+        else:
+            enum_str += f'{", " if not is_first_choice else ""}{choice_number}="{choice_value}"'
         is_first_choice = False
```

Comment:
I suppose this is slightly wrong because `choice_number` is not necessarily consecutive (?):
```suggestion
        for i, (choice_number, choice_value) in enumerate(signal.choices.items()):
            if i > 0:
                enum_str += ', '
            if i > 0 and i%10 == 0:
                enum_str += '\n'
            enum_str += f'{choice_number}="{choice_value}"'
```

Reply:
Thanks @andlaus that's a great point.

Reply:
I've added a new commit based on your suggestion on non-consecutive enum values @andlaus and a unit test for multiline enum sym files